cmd/zapi/cmd/post/pcap.go

```diff
@@ -12,63 +12,50 @@ import (
 	"time"
 
 	"github.com/brimsec/zq/api"
-	"github.com/brimsec/zq/api/client"
 	"github.com/brimsec/zq/cmd/zapi/cmd"
 	"github.com/brimsec/zq/cmd/zapi/format"
 	"github.com/brimsec/zq/pkg/display"
 	"github.com/mccanne/charm"
 )
 
-var PcapPost = &charm.Spec{
-	Name:  "pcappost",
-	Usage: "pcappost [options] path",
+var Pcap = &charm.Spec{
+	Name:  "postpcap",
```

Comment:
Nit: s/Pcap/PostPcap/ so the variable name matches the command.

Reply:
Good point, I'm going to change the rest of them for consistency as well.